packages/language-server/src/plugins/astro/AstroPlugin.ts

```diff
@@ -41,7 +41,7 @@ export class AstroPlugin implements Plugin {
 		return completions;
 	}
 
-	async getFoldingRanges(document: AstroDocument): Promise<FoldingRange[]> {
+	getFoldingRanges(document: AstroDocument): FoldingRange[] {
```

Comment:
Just a side-note, but we've started always adding a return signature in the main astro repo (informally), and I like to see that happening here as well. Think it helps with bugs.

Reply:
Not sure what you mean by a "return signature"? You mean a return type?